src/internal/operations/test/exchange_test.go

```diff
@@ -1092,9 +1126,24 @@ func (suite *ExchangeRestoreIntgSuite) TestRestore_Run_exchangeWithAdvancedOptio
 			GetItemsInContainerByCollisionKeyer[string](acCont),
 			collKeys[path.ContactsCategory])
 
+		currentContactIDs, err := acCont.GetItemIDsInContainer(ctx, userID, cIDs[path.ContactsCategory])
+		require.NoError(t, err, clues.ToCore(err))
+
+		assert.Equal(t, len(contactIDs), len(currentContactIDs), "count of ids ids are equal")
+		for orig := range contactIDs {
+			_, ok := currentContactIDs[orig]
+			assert.False(t, ok, "original item should not exist after replacement")
```

Comment:
you could maybe try `assert.NotContains` is you want. Not sure if the test output from that would be any clearer though

Reply:
Worked!